package.json

```diff
@@ -38,11 +38,13 @@
     "supertest": "^6.1.6"
   },
   "dependencies": {
+    "basic-server-backend": "file:",
```

Comment:
What is that supposed to do? can you please remove it?
```suggestion
```

Reply:
yes,It is removed now please check

Reply:
it still hasn't